rpc/grpc/client/privileged/pruning_service.go

```diff
@@ -16,17 +16,59 @@ type RetainHeights struct {
 	PruningService uint64
 }
 
+type RetainHeight struct {
+	Height uint64
+}
+
```

Comment:
I would not invent a struct type for just this one field, just return a `uint64` like in the other getters returning a scalar.
It's also awfully ambiguous with `RetainHeights` returned by `GetBlockRetainHeight` (which, I admit, was also ambiguously named).

Reply:
fixed